Enable building docs without conda

## Expected Behavior

I want to work on the docs - but I don't want to have to install conda to do that.

## Actual Behavior

```
/bin/bash: /home/mgielda/antmicro/github.com/skywater-pdk/env/conda/bin/activate: No such file or directory
make: *** [Makefile:79: html] Error 1
```

## Steps to Reproduce the Problem

```
cd docs
make html
```
I have a solution already, will do a PR in a sec.

--- a/docs/Makefile
+++ b/docs/Makefile
@@ -3,7 +3,13 @@
 DOCSDIR := $(dir $(lastword $(MAKEFILE_LIST)))
 TOPDIR := $(realpath $(DOCSDIR)/..)
 
+ifndef NO_CONDA
 include $(TOPDIR)/scripts/make/conda.mk
+endif
+ifdef NO_CONDA
+IN_CONDA_ENV :=
+CONDA_ENV_PYTHON :=
+endif
 
 SHELL = /bin/bash
 

